src/Main.java

```diff
@@ -0,0 +1,41 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Main {
+
+    public static void main(String[] args) {
+        ArrayList<Integer> list = new ArrayList<Integer>(List.of(11, 3, -3, -6, -2));
+
+        System.out.println("the list: " + list);
+        System.out.println("closest number to zero: "+ closestToZero(list));
+    }
+
+    //    this method returs the closet integer to 0 in a list
```

Comment:
https://i0.wp.com/css-tricks.com/wp-content/uploads/2017/10/DLKxcnZUQAAFQkI.jpg?ssl=1  // this is a bridge